controllers/places.js

```diff
@@ -0,0 +1,62 @@
+const Place = require('../models/Place');
+const passport = require('../config/passport');
+
+exports.listPlaces = async (req, res) => {
+  const places = await Place.find();
+  res.render('places/listPlaces', { places });
+};
+
+exports.addPlace = (req, res) => res.render('places/addPlace');
+
+exports.addPlaceProcess = async (req, res) => {
+  const { name, placeType, longitude, latitude } = req.body;
+  const location = {
+    type: 'Point',
+    coordinates: [longitude, latitude],
+  };
+  await Place.create({
```

Comment:
Aca solo te hizo falta realmente asignar el lugar creado a la persona que esta creando el lugar(que para el caso practico es el user en session) Para ello tuviste que hacer algo como: 

```suggestion
  const newPlace = await Place.create({
```

y...